client/src/partials/vouchers/simple.html

```diff
@@ -0,0 +1,144 @@
+<!-- breadcrumb header -->
+<bh-breadcrumb
+  path="SimpleVoucherCtrl.paths">
+</bh-breadcrumb>
+
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-md-6">
+
+        <div class="panel panel-primary">
+          <div class="panel-heading">
+            {{ "VOUCHERS.SIMPLE.TITLE" | translate }}
```

Comment:
If you could add translation keys at least for English it should speed up the process of getting everything translated into all required languages.